### DISCUSSION TITLE
How do I know which notes are Smart Connections's answer (through ChatGPT) based on?

### DISCUSSION THREAD

**Discussion by KelvinHYYip** [brianpetro/obsidian-smart-connections#400] [Q&A]
Am I missing something? Is there a way to know which notes does the plug-in refer to in the responses? Thank you.

**Comment by brianpetro** [comment]
In version 1 (current as of Dec 2023): In the bottom right corner of the message in the Smart Chat, there are two buttons. One copies the message, and the other copies the context used to generate the message. You can then paste this into a note for review.

Coming soon in version 2: the chat history will be saved in markdown format and links to notes used as context will be part of that.

**Reply by KelvinHYYip** [reply]
Thanks Brian! Somehow the "eye" icon at the bottom right corner of the message box did not show up. It is now there after refreshing obsidian. ;)
Excellent work!!
